fdbserver/Ratekeeper.actor.cpp

```diff
@@ -173,7 +181,10 @@ class RatekeeperImpl {
 			}
 		} catch (...) {
 			// including cancellation
-			self->storageQueueInfo.erase(myQueueInfo);
+			if (self.available()) {
```

Comment:
`!self.available()` here?

Reply:
This is intended, so we only operate on`self` when it is available, otherwise, SEGV might happen.